docs/website/docs/architecture.md

```diff
@@ -4,8 +4,23 @@ sidebar_position: 4
 
 # Architecture
 
+![architecture-diagram](/img/architecture-diagram.png)
+
```

Comment:
maybe a short introduction?
`dlt` is a python library that you import and use in your code. `dlt` will run in standalone scripts, notebooks, streamlit apps, airflow operators and is self sufficient - does not require any backend process to run along it. If you provide `dlt` with the data that you want to load and tell where it should go, it will take the data through the steps described below.